packages/design-system/src/styles/_base.scss

```diff
@@ -120,7 +120,8 @@ $breakpoints: (
   }
 }
 
-a,
+// ❓ if we style the <a> tag, what's the point of this link class?
```

Comment:
Only used in a couple components which overwrite colors (Alert), or extend it (UsaBanner)..